windows-nsis/openvpn.nsi

```diff
@@ -317,17 +327,16 @@ Section /o "${PACKAGE_NAME} GUI" SecOpenVPNGUI
 	SetOverwrite on
 	SetOutPath "$INSTDIR\bin"
 
-	File "${OPENVPN_ROOT}\bin\openvpn-gui.exe"
+	${If} ${RunningX64}
+		File "${OPENVPN_ROOT_X86_64}\bin\openvpn-gui.exe"
+	${Else}
+		File "${OPENVPN_ROOT_I686}\bin\openvpn-gui.exe"
+	${EndIf}
 
 	${If} ${SectionIsSelected} ${SecAddShortcutsWorkaround}
 		CreateDirectory "$SMPROGRAMS\${PACKAGE_NAME}"
 		CreateShortCut "$SMPROGRAMS\${PACKAGE_NAME}\${PACKAGE_NAME} GUI.lnk" "$INSTDIR\bin\openvpn-gui.exe" ""
 		CreateShortcut "$DESKTOP\${PACKAGE_NAME} GUI.lnk" "$INSTDIR\bin\openvpn-gui.exe"
-
-		# This is required because of commit 2bb1726764344 ("Do not disconnect on suspend").
-		# Without this users will experience weired disconnections after suspend/resume.
-		WriteRegStr "HKLM" "Software\OpenVPN-GUI" "disconnect_on_suspend" "0"
-
```

Comment:
Just a note for the posterity: we [agreed on the mailing list](https://www.mail-archive.com/openvpn-devel@lists.sourceforge.net/msg13127.html) to remove this registry string..